### DISCUSSION TITLE
How to use trame/paraview to achieve complex functions

### DISCUSSION THREAD

**Discussion by chiyandetaotie** [Kitware/trame#516] [Trame/ParaView]
How to use trame/paraview to achieve complex functions? I want to use trame to achieve complex functions, such as slicing and clipping unstructured grids. Can I only write code through python VTK to achieve it?I have checked the trame/paraview cases in the tutorials (https://kitware.github.io/trame/guide/tutorial/paraview.html) , but there is no detailed description of the interface of paraview and the use of other complex functions.How to achieve complex functions with the powerful paraview?

**Comment by jourdain** [comment]
The trame tutorial just cover how to display a view from ParaView. After that, you are free to use the ParaView API to do the processing you aim to do. But it can't be up to trame to provide the ParaView documentation. Instead, you should look at [the ParaView resources/documentation](https://www.paraview.org/resources/)
